src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -51,7 +51,7 @@ const createEmptyTable = (startDate, company) => {
     //TODO's: Pull UserID automatically
     return {
         UserID: "77566d69-3b61-452a-afe8-73dcda96f876",
-        TimesheetID: uuidv4(), 
+        TimesheetID: Math.round(Math.random() * 1000000000), 
```

Comment:
Why this change?

Reply:
Oh the type was a number whereas uuidv4 is a string so I just created very large number as a workaround for the moment I should probably actually fix this